package-system/Qt/docker_build_qt_linux.sh

```diff
@@ -69,7 +69,6 @@ echo Configuring Qt  ...
                  -no-feature-getentropy \
                  -no-feature-statx \
                  -feature-wayland-server \
```

Comment:
We should remove the wayland server feature as its under GPLv3 and we don't need it